__tests__/app.test.js

```diff
@@ -225,3 +226,75 @@ describe("POST /api/articles/:article_id/comments", () => {
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 119,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      });
+  });
+
+  test("should decrease vote count correctly if given negative int", () => {
+    const updatedVote = { inc_votes: -15 };
+
+    return request(app)
+      .patch("/api/articles/2")
+      .send(updatedVote)
+      .expect(201)
+      .then(({ body }) => {
+        const { update } = body;
+        expect(update).toEqual({
+          article_id: 2,
+          title: "Sony Vaio; or, The Laptop",
+          topic: "mitch",
+          author: "icellusedkars",
+          body: "Call me Mitchell. Some years ago—never mind how long precisely—having little or no money in my purse, and nothing particular to interest me on shore, I thought I would buy a laptop about a little and see the codey part of the world. It is a way I have of driving off the spleen and regulating the circulation. Whenever I find myself growing grim about the mouth; whenever it is a damp, drizzly November in my soul; whenever I find myself involuntarily pausing before coffin warehouses, and bringing up the rear of every funeral I meet; and especially whenever my hypos get such an upper hand of me, that it requires a strong moral principle to prevent me from deliberately stepping into the street, and methodically knocking people’s hats off—then, I account it high time to get to coding as soon as I can. This is my substitute for pistol and ball. With a philosophical flourish Cato throws himself upon his sword; I quietly take to the laptop. There is nothing surprising in this. If they but knew it, almost all men in their degree, some time or other, cherish very nearly the same feelings towards the the Vaio with me.",
+          created_at: "2020-10-16T05:03:00.000Z",
+          votes: -15,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      });
+  });
+  test("should give error when passed valid but non existing article ID ", () => {
+    const updatedVote = { inc_votes: 19 };
+
+    return request(app)
+      .patch("/api/articles/99")
+      .send(updatedVote)
+      .expect(404)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toEqual("Not Found");
+      });
+  });
+  test("should ", () => {
+    const updatedVote = { inc_votes: 1 };
+
+    return request(app)
+      .patch("/api/not-a-url/1")
+      .send(updatedVote)
+      .expect(400)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toEqual("Bad Request");
+      });
+  });
+});
```

Comment:
You can also add a test to check what happens if the updatedVote object has missing keys/incorrect data type passed to it!